config/locales/en.yml

```diff
@@ -22,6 +22,7 @@ en:
       submit: Submit
     user:
       create:
+        failed: The service is experiencing issues creating new users and the team has been alerted to the problem.
```

Comment:
There is nothing the user can do correct the issue if this scenario happens. The best we can do at this point (without a support desk) is to reassure them that we have noticed the Rollbar event and will be dealing with it.